backend/fuzz_tests/slack/contribute_test.py

```
@@ -0,0 +1,40 @@
+        issue_count,
+    ):
+        settings.SLACK_EVENTS_ENABLED = events_enabled
+        mock_active_projects_count.return_value = project_count
+        mock_open_issues_count.return_value = issue_count
+        mock_slack_event = {"user": "U123456", "channel": OWASP_CONTRIBUTE_CHANNEL_ID}
+        mock_slack_client = MagicMock()
+        mock_slack_client.conversations_open.return_value = {"channel": {"id": "C123456"}}
+
+        contribute_handler(event=mock_slack_event, client=mock_slack_client, ack=MagicMock())
+
+        assert True
```

Ruff (0.8.2)

Use of `assert` detected

(S101)